apps/application-system/api/src/app/modules/application/application.module.ts

```diff
@@ -65,6 +66,7 @@ if (process.env.INIT_SCHEMA === 'true') {
     SigningModule.register(environment.signingOptions),
     CmsTranslationsModule,
     LoggingModule,
+    NationalRegistryXRoadModule,
```

Comment:
Where is this used ?

Reply:
The `NationalResistryXRoadService` is used in the `Driving License Service.` Since the `Driving License application` needs to use that service the` application system `needs to have the `NationalRegistryXRoadModule` accessible.